irods/manager/data_object_manager.py

```diff
@@ -64,7 +66,7 @@ def should_parallelize_transfer( self,
         # Example: $ export IRODS_VERSION_OVERRIDE="4,2,9" ;  python -m irods.parallel ...
         # ---
         # Delete the following line on resolution of https://github.com/irods/irods/issues/5932 :
-        if getattr(self.sess,'ticket__',None) is not None: return False
+        if self.sess.ticket__: return False
```

Comment:
Tell me about this.

Reply:
Because of irods/irods#5932, parallel put/get can't be combined with tickets.  So we abort early.  It used to be ticket__ was only conditionallly defined for session objects, but as of this PR I forced it to be an empty string at `__init__` time... therefore the getattr is not required.